src/core/Expander/Expander/Expander.md

```diff
@@ -11,6 +11,22 @@ import {
 </Expander>;
 ```
 
+```jsx
+import {
+  Expander,
+  ExpanderTitle,
+  ExpanderContent,
+  Checkbox
+} from 'suomifi-ui-components';
+
+<Expander className="expander-test" defaultOpen={true}>
+  <ExpanderTitle>
```

Comment:
As the `ariaCloseText` and `ariaOpenText` props are required, it might be good idea to add them to the example.

Reply:
😅  Why bother?

Reply:
Fixed.

Reply:
Note that these props are only mandatory for ExpanderTitle, not ExpanderTitleButton.